voicevox_engine/setting/SettingLoader.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+
+import yaml
+
+from .Setting import Setting
+
+
+class SettingLoader:
+    def __init__(self, setting_file_path: Path) -> None:
+        self.setting_file_path = setting_file_path
+
+    def load_setting_file(self) -> Setting:
+        setting = yaml.safe_load(self.setting_file_path.read_text(encoding="utf-8"))
+
+        setting = Setting(
+            cors_policy_mode=setting["cors_policy_mode"],
+            allow_origin=setting["allow_origin"],
+        )
+
+        return setting
+
+    def dump_setting_file(self, settings: Setting) -> None:
+        settings_dict = settings.dict()
+
+        with open("setting.yml", "w") as file:
+            yaml.dump(settings_dict, file)
```

Comment:
上でutf-8で読み込むようになっているので、書き込みもutf-8を強制した方が良いかと思います。
また、open関数を使っている他の場所では、`with open() as f`と書いているので、それに合わせるときれいかなと思います。
```suggestion
        with open("setting.yml", mode="w", encoding="utf-8") as f:
            yaml.dump(settings_dict, f)
```